test/snapshot/__snapshots__/class.test.js.snap

```diff
@@ -1158,163 +1173,7 @@ Program {
-              "nullable": TypeReference {
-                "kind": "typereference",
-                "name": "int",
-                "raw": "int",
-              },
-              "readonly": false,
-              "type": [],
-              "value": null,
-            },
-          ],
-          "visibility": "",
-        },
-      ],
-      "extends": null,
-      "implements": null,
-      "isAbstract": false,
-      "isAnonymous": false,
-      "isFinal": false,
-      "isReadonly": false,
-      "kind": "class",
-      "name": Identifier {
-        "kind": "identifier",
-        "name": "Bob",
-      },
-    },
-    ExpressionStatement {
-      "expression": undefined,
-      "kind": "expressionstatement",
-    },
-  ],
-  "errors": [
-    Error {
-      "expected": undefined,
-      "kind": "error",
-      "line": 3,
-      "message": "readonly properties can be used only on class constructor on line 3",
-      "token": undefined,
-    },
-    Error {
-      "expected": 222,
-      "kind": "error",
-      "line": 3,
-      "message": "Parse Error : syntax error, unexpected 'readonly' (T_READ_ONLY), expecting T_VARIABLE on line 3",
-      "token": "'readonly' (T_READ_ONLY)",
-    },
-    Error {
-      "expected": [
-        ",",
-        ")",
-      ],
-      "kind": "error",
-      "line": 3,
-      "message": "Parse Error : syntax error, unexpected 'readonly' (T_READ_ONLY) on line 3",
-      "token": "'readonly' (T_READ_ONLY)",
-    },
-    Error {
-      "expected": ")",
-      "kind": "error",
-      "line": 3,
-      "message": "Parse Error : syntax error, unexpected 'readonly' (T_READ_ONLY), expecting ')' on line 3",
-      "token": "'readonly' (T_READ_ONLY)",
-    },
-    Error {
-      "expected": "{",
-      "kind": "error",
-      "line": 3,
-      "message": "Parse Error : syntax error, unexpected 'readonly' (T_READ_ONLY), expecting '{' on line 3",
-      "token": "'readonly' (T_READ_ONLY)",
-    },
-    Error {
-      "expected": [
-        ",",
-        ";",
-        "=",
-      ],
-      "kind": "error",
-      "line": 3,
-      "message": "Parse Error : syntax error, unexpected ')' on line 3",
-      "token": "')'",
-    },
-    Error {
-      "expected": ";",
-      "kind": "error",
-      "line": 3,
-      "message": "Parse Error : syntax error, unexpected ')', expecting ';' on line 3",
-      "token": "')'",
-    },
-    Error {
-      "expected": [
-        198,
-        222,
-        182,
-      ],
-      "kind": "error",
-      "line": 3,
-      "message": "Parse Error : syntax error, unexpected '{' on line 3",
-      "token": "'{'",
-    },
-    Error {
-      "expected": "EXPR",
-      "kind": "error",
-      "line": 4,
-      "message": "Parse Error : syntax error, unexpected '}' on line 4",
-      "token": "'}'",
-    },
-  ],
-  "kind": "program",
-}
-`;
+exports[`Test classes Test that readonly method parameters are throwing errors 1`] = `"readonly properties can be used only on class constructor on line 3"`;
```

Comment:
This looks like a regression.

Reply:
This is related to the change you commented in the class.test file https://github.com/glayzzle/php-parser/pull/1143/files#r1865229710

Instead of letting the parser getting lost and generate gibberish errors, the exception is fired and serialized, using `toThrowErrorMatchingSnapshot`.